source/common/json/config_schemas.cc

```diff
@@ -670,6 +670,13 @@ const std::string Json::Schema::ROUTE_ENTRY_CONFIGURATION_SCHEMA(R"EOF(
       "host_redirect" : {"type" : "string"},
       "path_redirect" : {"type" : "string"},
       "prefix_rewrite" : {"type" : "string"},
+      "regex_rewrite" : {
```

Comment:
No need for JSON config schemas.. this is all deprecate, I'm surprised we haven't deleted it yet.